src/java.desktop/windows/classes/sun/awt/windows/WMenuItemPeer.java

```diff
@@ -146,20 +144,15 @@ public void run() {
     private static Font defaultMenuFont;
 
     static {
-        defaultMenuFont = AccessController.doPrivileged(
-            new PrivilegedAction <Font> () {
-                public Font run() {
-                    try {
-                        ResourceBundle rb = ResourceBundle.getBundle("sun.awt.windows.awtLocalization");
-                        return Font.decode(rb.getString("menuFont"));
-                    } catch (MissingResourceException e) {
-                        if (log.isLoggable(PlatformLogger.Level.FINE)) {
-                            log.fine("WMenuItemPeer: " + e.getMessage()+". Using default MenuItem font.", e);
-                        }
-                        return new Font("SanSerif", Font.PLAIN, 11);
-                    }
-                }
-            });
+        try {
```

Comment:
I guess something else (logging?) needs that SuppressWarning you left on the class ??

Reply:
no, removed..